src/azure-cli/azure/cli/command_modules/rdbms/_help.py

```diff
@@ -878,3 +878,27 @@
   - name: List log files for 'testsvr' less than 30Kb in size.
     text: az postgres server-logs list -g testgroup -s testsvr --max-file-size 30
 """
+
+helps['postgres get-access-token'] = """
```

Comment:
I suggest don't do this; rather just redirect users to use the generic `az account -get-access-token`, and you can even add an example over there.

Reply:
I see that az account get-access-token will work if --resource is specified. My question is how do I redirect users over there, and what is the appropriate way to show an example of looking up the RDBMS endpoint in "show az cloud", and using that in the command? Is it really that bad to just add a separate command to make it easier for the customer?

Reply:
I kept the rdbms get-access-token commands, but implemented the same as az account get-access-token, plus updated the help documentation to indicate that these commands are equivalent to az account get-access-token with --resource set to the ossrdbmsEndpointId from az cloud show. Let me know if this is enough.

Reply:
looking up the RDBMS endpoint in "show az cloud"
You had a good point here.

2 reasons that I suggest we don't introduce new commands on this
1. The rule of thumb is we should only have one command to do one specific thing
2. If we expose these 3 commands, for consistency, we need to do for any commands with unique resource such as `az storage`, `az ad`, `az dls`, etc

For the best of both world, maybe we can update the `--resource` argument `az account get-access-token` to accept an enum values such as `arm`, `rdbms`, `aad_graph`, `ms_graph`, `data_lake`, `storage`, etc?

Reply:
I like this idea (allow enums for --resource). Let me work on it. If that's implemented, then I'm happy to remove the rdbms-specific commands.

Reply:
I have made the suggested changes. Please review and advise whether or not the implementation is acceptable. Thanks!

Reply:
My attempt at an upstream merge worked, but for some reason, github wants me to create a new pull request. I am going to abandon this one. The new PR is https://github.com/Azure/azure-cli/pull/10091 .